odxtools/dopbase.py

```diff
@@ -20,18 +24,21 @@ class DopBase(IdentifiableElement):
     Any class that a parameter can reference via a DOP-REF should inherit from this class.
     """
 
-    is_visible_raw: Optional[bool]
```

Comment:
Why was `is_visible_raw` removed?

Reply:
because DOP-BASE does not feature such an attribute...